xapian-core/queryparser/termgenerator_internal.cc

```diff
@@ -121,9 +121,9 @@ inline unsigned check_suffix(unsigned ch) {
 }
 
 // FIXME: add API for this:
```

Comment:
You shouldn't commit commented-out code; just remove it (and the above comment). git makes it easy enough to figure out what was there before if we ever need it.